doc-content/enterprise-only/installation/installer-run-proc.adoc

```diff
@@ -141,15 +142,15 @@ A data source is an object that enables a Java Database Connectivity (JDBC) clie
 . Click *Next* to configure the runtime environment.
 endif::[]
 . When *Processing finished* appears at the top of the screen, click *Next* to complete the installation.
-. If desired, click *Generate Installation Script and Properties File* to save the installation data in an XML file, and then click *Done*. You can use this file to automatically install {PRODUCT} on the same type of server. Note that you must change the `installpath` parameter in the XML file to specify the path of the new server that you want to install {PRODUCT} on. Enter the following command to perform an installation with the XML file:
+. If desired, click *Generate Installation Script and Properties File* to save the installation data in the `auto.xml` file, and then click *Done*. You can use this file to automatically install {PRODUCT} on the same type of server. Note that you must change the `installpath` parameter in the XML file to specify the path of the new server that you want to install {PRODUCT} on. Enter the following command to perform an installation with the XML file:
```

Comment:
Same here,  I removed first two lines (difference from previous comment):
_If desired, click *Generate Installation Script and Properties File* to save the installation data in the `auto.xml` file, and then click *Done*_
The XML file allows you to easily repeat {PRODUCT} installation on the same type of server with same configuration on multiple machines. You can use the click-through installer to generate this XML file once and then easily repeat the installation on different machines. It can also be used it on the same machine, but you must change the <installpath> parameter in the XML file to point towards new server.
Enter the following command to perform an installation with the XML file: